packages/dds/tree/src/feature-libraries/editable-tree/editableTreeContext.ts

```diff
@@ -225,6 +309,20 @@ export class ProxyContext implements EditableTreeContext {
         });
     }
 
+    public replaceNodes(
+        path: UpPath | undefined,
+        fieldKey: FieldKey,
+        index: number,
+        count: number,
+        newContent: ITreeCursor | ITreeCursor[],
+    ): boolean {
+        return this.runTransaction((editor) => {
+            const field = editor.sequenceField(path, fieldKey);
+            field.delete(index, count);
+            field.insert(index, newContent);
+        });
+    }
```

Comment:
Not sure if it should be mentioned in the EditableTree/EditableField API documentation, how this works exactly in a concurrent environment. Should we?